recipes/pals-schema/recipe.yaml

```diff
@@ -0,0 +1,59 @@
+context:
+  name: pals-schema
+  version: "0.1.1"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/p/pals_schema/pals_schema-${{ version }}.tar.gz
+  sha256: 661925959212186fc100898d5f814985b829e047e93bf08b43ac4d8720d7532e
+
+build:
+  noarch: python
+  script: python -m pip install . -vv
```

Comment:
```suggestion
  script: python -m pip install . --no-build-isolation --no-deps -vv
```

Reply:
Thanks! Oh, I think the CF KB is outdated then :)